content/decomposed/campaigns/placeholder.txt

```diff
@@ -0,0 +1 @@
+<to be removed once the automation is in place>
```

Comment:
I don't think we need these.  The new directories can be added as part of getting the decomposition running

Reply:
Alright, removed them